src/herbivoresAndCarnivores.js

```diff
@@ -1,15 +1,36 @@
 'use strict';
 
 class Animal {
-  // write your code here
+  constructor() {
+    this.name = '';
```

Comment:
constructor should accept at least `name`. Because this way all of your animal will have `name = ''`